app/views/application/index.html.erb

```diff
@@ -1,309 +1,69 @@
-      popupAnchor: [1, -34],
-      shadowSize: [41, 41]
-    });
-    var blueIcon = new L.Icon({
-      iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-blue.png',
-      shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
-      iconSize: [25, 41],
-      iconAnchor: [12, 41],
-      popupAnchor: [1, -34],
-      shadowSize: [41, 41]
-    });
-    var yellowIcon = new L.Icon({
-      iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-yellow.png',
-      shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
-      iconSize: [25, 41],
-      iconAnchor: [12, 41],
-      popupAnchor: [1, -34],
-      shadowSize: [41, 41]
-    });
-
-    var bird_data = null;
-    var bird_images = null;
-    var mymap = L.map('mapid').setView([37.42, -121.91], 13);
-    L.tileLayer('https://{s}.tile.thunderforest.com/outdoors/{z}/{x}/{y}.png?apikey={apikey}', {
-        attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-        apikey: 'df20324a5586441197c9410773b4ce60',
-        maxZoom: 22
-     }).addTo(mymap);
-
-    var legend = L.control({position: 'topright'});
-    legend.onAdd = function (map) {
-      var div = L.DomUtil.create('div', 'legend');
-      div.innerHTML +=  '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png" height=41 width=25>'  + 'Your Location' + '<br>'
-      div.innerHTML +=  '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-gold.png" height=41 width=25>' + 'Bird-Watching Park' + '<br>'
-      div.innerHTML +=  '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-blue.png" height=41 width=25>'   +  'Recent Bird Sighting'
-      div.style.background = "silver"
-      return div;
-    };
-    legend.addTo(mymap);
-
-    function setContent(birddata,imgsrc,elem){
-      $(elem).empty();
-      $(elem).append(
-        `
-        <img class="card-img-top" src="${imgsrc}">
-        <div class="card-body">
-          <h5 class="card-title">${birddata.comName}</h5>
-          <h6 class="card-subtitle mb-2 text-muted">${birddata.sciName}</h6>
-          <p class="card-text">${birddata.distTo} miles away, at ${birddata.locName}</p>
-          <p class="card-text">reported ${ moment(birddata.obsDt).fromNow()  }</p>
-        </div>
-        <a href="https://ebird.org/species/${birddata.speciesCode}" class="btn btn-primary">View On Ebird</a>
-        `
-      );
-    }
-
-
-
-    function setContentLoc(i, birddata,imgsrc,elem){
-      $(elem)[i].innerHTML +=
-        `
+    <div class="row">
+      <div class="col-md-2"></div>
+      <div class="col-md-8">
+        <div id="RecentBirdSightings">
```

Comment:
The CSS ID style seems to be all over the place- this one uses TitleCase, another used snake_case, and another used hyphen-ated. I suggest picking one and sticking to it if possible!

Reply:
I tried fixing this, but I'm not sure how the JS works for rendering the bird images and this seems to affect the functionality of the app, so I'm leaving this until we can figure this out. Hope that's ok!

Reply:
Yep, totally fine.